ons_alpha/articles/models.py

```diff
@@ -45,7 +46,15 @@ class ArticlePage(BundledPageMixin, RoutablePageMixin, BasePage):
     updates = StreamField(CorrectionsNoticesStoryBlock(), blank=True, use_json_field=True)
 
     content_panels = (
-        BasePage.content_panels
+        [
+            FieldPanel("title"),
+            FieldPanel(
+                "headline",
+                help_text="Use this as a news headline. When set, replaces the title on the page. "
+                "Note that the page <code>slug</code> is driven by the title field.",
```

Comment:
Nit: They can be changed independently. Is this something which needs flagging here explicitly?

Reply:
I am not sure. Let's get feedback and see :)